src/commands/fun/piada.js

```diff
@@ -0,0 +1,21 @@
+const { sample } = require('lodash');
+const { client } = require('../../core/twitch_client');
+const piadas = require('../../utilities/data/piadas.json');
+const wait = require('../../utilities/wait');
+
+const SECONDS_TO_ANSWER = 3;
+
+module.exports = {
+  keyword: 'piada',
+  async execute({ channel }) {
+    if (piadas.length === 0) {
+      return;
+    }
+
+    const piada = sample(piadas);
```

Comment:
Acho que por hora é desnecessário colocar o lodash como dependência do projeto.
Pode usar as funções de `Math` para pegar um item aleatório do Array:

```suggestion
    const piada= piadas[Math.floor(Math.random() * piadas.length)]
```

Reply:
Eu acho mais semântico ler `sample` do que uma expressão como esta, pensei em colocar o `lodash` pq ja vi nas issues que tem outras features que também vão precisar utilizar o mesmo mecanismo.

Reply:
Entendo.

Se a questão é legibilidade, pode criar uma função utilitária para abstrair esse uso das funções de `Math` e obter um item aleatório do Array. Minha revisão sobre o lodash é que, dado o tanto de outras funções que ele traz consigo, é desnecessário para o projeto no momento - boa parte das funções podem ser implementadas nativamente.

- https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#voice-of-developers

Sobre a `sample`, é basicamente a mesma coisa da sugestão com uma checagem de tamanho de string antes: https://github.com/lodash/lodash/blob/2f79053d7bc7c9c9561a30dda202b3dcd2b72b90/sample.js#L15